initialize-project

```diff
@@ -69,4 +73,4 @@ git_clone jspecify --no-single-branch
 
 git_clone jdk --depth 1 --single-branch
 
-git_clone checker-framework
+git_clone checker-framework --depth 1 --single-branch
```

Comment:
The point of `$SHALLOW` is to allow CIs like GitHub actions to avoid cloning any commits they don't care about. If you're just running tests for CI you don't need to check any past commits out.

I think if someone has their own fork of eisop we wouldn't want them to clone only one branch. I would suggest continuing to rely on `$SHALLOW` (defined externally) for `checker-framework`.

Reply:
Not really related to this PR, but:

> [Use `--filter=blob:none` instead of `--depth 1`](https://github.com/typetools/jdk/commit/6c6241dc47453a3b1bccda4c7dff4df1795241c1)

I wonder if that's worth investigating someday.

Reply:
I had a quick look at this post:
https://github.blog/2020-12-21-get-up-to-speed-with-partial-clone-and-shallow-clone/

It seems to generally prefer the blob-less mode. For CI, where one only has a single build, continuing to use `--depth 1` seems okay. But as this script is for developer set-up, using blob-less might be better.

Reply:
Let's postpone that to a different PR. It's not related to switching to EISOP.